mix/mix_protocol.nim

```diff
@@ -153,20 +157,24 @@ proc handleMixNodeConnection(
           await nextHopConn.close()
         except CatchableError as e:
           error "Failed to close outgoing stream: ", err = e.msg
+  # TODO?: contextualise why this is being discarded.
   of Duplicate:
     discard
   of InvalidMAC:
     discard
 
 proc anonymizeLocalProtocolSend*(
     mixProto: MixProtocol,
+    # TODO: this should be a distinct message type
```

Comment:
primitive data types used in a specific way should be retyped. This helps avoud confusion and misuse.